lnwallet/channel.go

```diff
@@ -3961,3 +3961,8 @@ func CreateCooperativeCloseTx(fundingTxIn *wire.TxIn,
 func (lc *LightningChannel) CalcFee(feeRate uint64) uint64 {
 	return (feeRate * uint64(commitWeight)) / 1000
 }
+
+// RemoteNextRevocation returns the channelState's RemoteNextRevocation.
+func (lc *LightningChannel) RemoteNextRevocation() *btcec.PublicKey {
+	return lc.channelState.RemoteNextRevocation
```

Comment:
This method should be holding on to the mutex embedded within this struct. Otherwise, a race condition can arise between this method and `ReceiveRevocation` or `SignNextCommitment`.